packages/core/src/scss/components/checkout/paymentProvider/_paymentProvider.scss

```diff
@@ -5,17 +5,24 @@
     align-items: start;
     display: flex;
     justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.paymentProviderHeader-subtitleContainer {
+    flex-basis: 100%;
```

Comment:
This line forces `subtitleContainer` to a new row.
My concern is that we don't want to split `nameContainer` and `cc` evenly on the first row.
`space-between` is still our ideal layout.
Therefore, could you please remove `flex-basis: 50%;` in the PR ?